dev_guide/builds.adoc

```diff
@@ -33,17 +33,22 @@ link:#docker-strategy-options[options])
 (link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[description],
 link:#custom-strategy-options[options])
 
-And there are three types of build source:
+And there are four types of build source:
 
 - link:#source-code[Git]
 - link:#dockerfile-source[Dockerfile]
+- link:#image-source[Image]
 - link:#binary-source[Binary]
 
 It is up to each build strategy to consider or ignore a certain type of source,
 as well as to determine how it is to be used.
 
 Binary and Git are mutually exclusive source types, while Dockerfile can be used
-by itself or together with Git and Binary.
+by itself or together with Git and Binary.  Image is intended to together with the
```

Comment:
to be fair i haven't confirmed in the code myself, but logically it should not require any other inputs, so i'd consider it a bug if it does.